contracts/ERC677BridgeToken.sol

```diff
@@ -8,17 +8,29 @@ import "./interfaces/IBurnableMintableERC677Token.sol";
 import "./upgradeable_contracts/Claimable.sol";
 
 contract ERC677BridgeToken is IBurnableMintableERC677Token, DetailedERC20, BurnableToken, MintableToken, Claimable {
-    address public bridgeContract;
+    address[] internal _bridgeContracts;
+    mapping(address => bool) internal _isBridgeContract;
 
     event ContractFallbackCallFailed(address from, address to, uint256 value);
 
     constructor(string _name, string _symbol, uint8 _decimals) public DetailedERC20(_name, _symbol, _decimals) {
         // solhint-disable-previous-line no-empty-blocks
     }
 
-    function setBridgeContract(address _bridgeContract) external onlyOwner {
-        require(AddressUtils.isContract(_bridgeContract));
-        bridgeContract = _bridgeContract;
+    function setBridgeContracts(address[] _contracts) external onlyOwner {
```

Comment:
The new `setBridgeContracts` function assumes that we just rewrite the list of bridges.

For example, we have three bridges: `[0x123, 0x456, 0x789]`. And then we want to remove the second one (0x456). We just pass the new array to the `setBridgeContracts`: [0x123, 0x789].

Reply:
Ok. Got you. Does it look like that we need to delete element here in order to get gas refund?
```
for (i = 0; i < _bridgeContracts.length; i++) {
    _isBridgeContract[_bridgeContracts[i]] = false;
}
```

Imagine that we are completely overwriting the array and replacing  `[0x123]` by `[0x321]`? even for the case when the array `[0x567, 0x789]` is overwritten by  `[0x345, 0x789]` it is still looks attractive.

Reply:
Do you mean using `delete _isBridgeContract[_bridgeContracts[i]];` instead of `_isBridgeContract[_bridgeContracts[i]] = false;`. I guess, these two statements are the same: https://solidity.readthedocs.io/en/v0.4.24/types.html#delete